checka11y.css

```diff
@@ -65,3 +65,12 @@ nav + nav::after {
   content: "ERROR: <nav> denotes the primary navigation for the page. Other navigations are permitted but should not be wrapped in a <nav>" !important;
   font-family: var(--checka11y-font, cursive);
 }
+
+/**
+ * Links
+ */
+
```

Comment:
I think we also need:

```css
a[target="_blank"] {
  border: var(--checka11y-error-border-width, 5px) solid var(--checka11y-error-color, red) !important;
}
```

I'm working on another branch to add different colours for warnings, but using the error colour is ok for now.